.github/azure-autodoc.yml

```diff
@@ -29,12 +29,17 @@ steps:
     targetType: inline
     workingDirectory: docs/
     script: |
+      echo "Diffing the generated docs with the previous state"
       git diff --quiet HEAD || \
       {
+        echo "Documentation needs updated"
+        git fetch
+        git checkout $(System.PullRequest.SourceBranch)
```

Comment:
hat das push upstream nicht gereicht? Ich denke das klappt so aber es kann ja auch meckern, dass du uncommited changes hast oder?

Reply:
Ich glaube solange sich auf dem branch in der zeit nichts geändert hat gibt es keine Konflikte

Reply:
Das upstream hat komischer Weise einen Commit erzeugt, der auf keinen Branch ist lauf github
![image](https://user-images.githubusercontent.com/12283268/88379698-b055d580-cda3-11ea-8fcb-601d5d35a0c2.png)